cmd/sink/filters/filters.go

```diff
@@ -49,17 +49,16 @@ func NewFilters(clientset kubernetes.Interface) *Filters {
 		delete:          make(map[NamespaceGroupVersionKind]cel.Program),
 		archiveOnDelete: make(map[NamespaceGroupVersionKind]cel.Program),
 	}
-
 }
 
 // getGlobalCelExprs returns three maps that have GroupVersionKind strings as keys and cel expression strings as values.
```

Comment:
need to update the doc for this function

```suggestion
// getGlobalCelExprs returns three maps that have GroupVersionKind as keys and cel expression strings as values.
```

Reply:
Aha!